sunpy/map/map_factory.py

```diff
@@ -79,6 +78,17 @@ class MapFactory(BasicRegistrationFactory):
 
     >>> mymap = sunpy.map.Map(data, header)   # doctest: +SKIP
 
+    * data, wcs object, in tuple
+    >>> from astropy.wcs import WCS
+    >>> wcs = WCS(AIA_171_IMAGE)     # doctest: +REMOTE_DATA
+    >>> mymap = sunpy.map.Map((data, wcs))    
+    
+    * data, wcs object, not in tuple
+    
+    >>> from astropy.wcs import WCS
+    >>> wcs = WCS(AIA_171_IMAGE)     # doctest: +REMOTE_DATA
+    >>> mymap = sunpy.map.Map(data, wcs)    
```

Comment:
Another # doctest: +REMOTE_DATA , we have a bug/hack with remote_data right now which needs this on any line that uses data that comes from our sample data.